### PR TITLE
Use errorReply instead of sendReply in necessary cases

### PR DESCRIPTION
**Pull request by Bryan-0**

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -603,7 +603,7 @@ exports.commands = {
 	roomdesc: function (target, room, user) {
 		if (!target) {
 			if (!this.runBroadcast()) return;
-			if (!room.desc) return this.sendReply("This room does not have a description set.");
+			if (!room.desc) return this.errorReply("This room does not have a description set.");
```
Comment (Slayer95) [inline commands.js]: This isn't an error, is it?

File: commands.js [hunk 2]
```diff
@@ -636,7 +636,7 @@ exports.commands = {
 	roomintro: function (target, room, user) {
 		if (!target) {
 			if (!this.runBroadcast()) return;
-			if (!room.introMessage) return this.sendReply("This room does not have an introduction set.");
+			if (!room.introMessage) return this.errorReply("This room does not have an introduction set.");
```
Comment (Slayer95) [inline commands.js]: Ditto

File: commands.js (hunk 1 above)
Comment (Bryan-0) [inline commands.js]: https://github.com/Zarel/Pokemon-Showdown/blob/master/commands.js#L676

That was before I made the PR, so I just realized that was an error too.

Comment (Zarel) [inline commands.js]: Yeah, these two should be reverted.

**Pull request merged**